tools/project_manager/page_compare.php

```diff
@@ -0,0 +1,203 @@
+<?php
+$relPath="./../../pinc/";
+include_once($relPath.'base.inc');
+include_once($relPath.'misc.inc');
+include_once($relPath.'theme.inc');
+include_once($relPath.'Project.inc');
+include_once($relPath."DPage.inc"); // remove_formatting()
+
+require_login();
+
+$L_round_options = array('P1', 'P2', 'P3', 'F1');
+$R_round_options = array('F1', 'F2');
+
+$projectid = validate_projectID('project', @$_GET['project']);
+$L_round_id = get_enumerated_param($_GET, "L_round_id", "P3", $L_round_options);
+$R_round_id = get_enumerated_param($_GET, "R_round_id", "F1", $R_round_options);
+$page_set = get_enumerated_param($_GET, "page_set", "all", array('left', 'right', 'all'));
+$go_compare = isset($_GET['compare']);
+
+$project = new Project($projectid);
+
+$state = $project->state;
+$title = $project->nameofwork;
+$sub_title = _('Compare pages with formatting removed');
+
+output_header("$sub_title: $title", NO_STATSBAR);
+
+echo "<h1>$title</h1>\n";
```

Comment:
I know other pages put the name of the work in the `<h1>` but it's really not helpful. This page isn't focused on the project, it's focused on comparing pages. I would swap these and put "Compare pages with formatting removed" as the `<h1>` and the nameofwork as the `<h2>`.